src/components/Gif/Gif.styled.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const MasonryItem = styled.div`
+  display: flex;
+  align-items: center;
+  flex: auto;
+  min-width: 150px;
+  margin: 0 8px 8px 0;
+  width: ${(props) => props.fullWidth ? '100% !important' : 'initial'};
```

Comment:
avoid using `important`. Not needed unless you are looking to override inline styling or library styles